packages/utils/package.json

```diff
@@ -26,9 +26,12 @@
   },
   "dependencies": {
     "@babel/generator": "^7.18.2",
-    "@babel/parser": "^7.18.5"
+    "@babel/parser": "^7.17.0",
```

Comment:
What's the reasoning for not choosing the latest version?

Reply:
This was an attempt to fix the types, but to no avail. Will bump them to the latest version.